src/data/users.tsx

```diff
@@ -337,6 +337,15 @@ const Users: User[] = [
     author: 'https://bsky.app/profile/did:plc:p2cp5gopk7mgjegy6wadk3ep',
     tags: ['client', 'favorite', 'opensource'],
   },
+  {
+    title: 'Skywalker',
+    description: 'Android client for Bluesky',
+    preview: require('./showcase/skywalker.png'),
+    website: 'https://bsky.app/profile/skywalkerapp.bsky.social',
+    source: 'https://github.com/mfnboer/skywalker',
+    author: 'https://bsky.app/profile/did:plc:qxaugrh7755sxvmxndcvcsgn',
+    tags: ['client'],
```

Comment:
A suggestion: as Skywalker is open-source, you might want to add the 'opensource' tag here as well :)